LightweightCharts/Classes/LightweightChartsModels/SeriesType.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+
+public enum SeriesType: String {
+    case line = "Line"
+    case area = "Area"
+    case candlestick = "Candlestick"
+    case bar = "Bar"
+    case histogram = "Histogram"
+    case none
```

Comment:
There is no need to add this case.
`JavaScriptEvaluator.decodedResult(forScript:, completion:)` will return `nil` on decoding error